File: app/src/main/java/com/fidoo/user/interfaces/AdapterCartAddRemoveClick.java
package com.fidoo.user.interfaces;


public interface AdapterCartAddRemoveClick {

    void onAddItemClick(String productId, String quantity, String offerPrice, String customizeid, String prodcustCustomizeId, String cart_id);
    void onRemoveItemClick(String productId, String quantity, String customizeid, String prodcustCustomizeId, String cart_id);
}

File: app/src/main/java/com/fidoo/user/restaurants/listener/AdapterCartAddRemoveClick.kt
package com.fidoo.user.restaurants.listener

 interface AdapterCartAddRemoveClick {
    fun onAddItemClick(
        quantity:String?,
        productId: String?,
        items: String?,
        offerPrice: String?,
        customizeid: String?,
        prodcustCustomizeId: String?,
        cart_id: String?
    )

    fun onRemoveItemClick(
        productId: String?,
        quantity: String?,
        customizeid: String?,
        prodcustCustomizeId: String?,
        cart_id: String?
    )
}